tests/client/disco/components/TestAddon.js

```diff
@@ -224,10 +225,28 @@ describe('<Addon />', () => {
       assert.ok(themeAction.calledWith(themeImage, THEME_RESET_PREVIEW));
     });
 
-    it('runs preventDefault onClick', () => {
+    it('installs a theme when the theme image is clicked', () => {
       const preventDefault = sinon.stub();
-      Simulate.click(themeImage, {preventDefault});
+      const installTheme = sinon.stub();
+      const data = { ...result, type: THEME_TYPE, themeAction,
```

Comment:
A little while back [1] it came up in review to add spaces when using destructuring. I guess this clearly shows we need to define when and when we aren't formatting objects with spaces and presumably if this is only in certain conditions e.g. was that first case just for a signature?

I'm going to take a look at what eslint can do for us here.

[1] https://github.com/mozilla/addons-frontend/pull/513#discussion_r65728712